projects/igniteui-angular/src/lib/grids/selection/selection.service.ts

```diff
@@ -72,7 +72,7 @@ export class IgxRow {
         return args;
     }
 
-    createDoneEditEventArgs(cachedRowData: any): IGridEditDoneEventArgs {
+    createPostCommitEditEventArgs(cachedRowData: any): IGridEditDoneEventArgs {
```

Comment:
If those are used for the exit event "post commit" is not quite accurate TBH, since exit can fire without commit too. The "done" naming was kind of okay.